core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -656,12 +659,22 @@ public Map<String, SaltPillarProperties> createPillarWithClouderaManagerSettings
                                 "missed_heartbeat_interval", cmMissedHeartbeatInterval,
                                 "disable_auto_bundle_collection", disableAutoBundleCollection,
                                 "set_cdp_env", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_0_2),
+                                "cloud_provider_setup_supported", isCloudProviderSetupSupported(stackDto, cmVersion),
+                                "cloud_provider", stackDto.getCloudPlatform(),
                                 "deterministic_uid_gid", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_2_1),
                                 "cloudera_scm_sudo_access", CMRepositoryVersionUtil.isSudoAccessNeededForHostCertRotation(clouderaManagerRepo)),
                         "mgmt_service_directories", serviceLocations.getAllVolumePath(),
                         "address", primaryGatewayConfig.getPrivateAddress()))));
     }
 
+    private boolean isCloudProviderSetupSupported(StackDto stackDto, String cmVersion) {
+        final List<String> supportedCloudProviders = Arrays.asList(CloudPlatform.AWS.name(), CloudPlatform.GCP.name(), CloudPlatform.AZURE.name());
+        if (supportedCloudProviders.contains(stackDto.getCloudPlatform()) && isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_6_2)) {
```

Comment:
Simplify please. If the return value depends on the condition then you can return the evaluated condition.

Reply:
The outmost parentheses are not required.